src/SPC/util/LicenseDumper.php

```diff
@@ -45,46 +45,58 @@ public function dump(string $target_dir): bool
     {
         // mkdir first
         if (is_dir($target_dir) && !FileSystem::removeDir($target_dir)) {
-            logger()->warning('Target dump directory is noe empty, be aware!');
+            logger()->warning('Target dump directory is not empty, be aware!');
         }
         FileSystem::createDir($target_dir);
         foreach ($this->exts as $ext) {
             if (Config::getExt($ext, 'type') !== 'external') {
                 continue;
             }
+
             $source_name = Config::getExt($ext, 'source');
-            $content = $this->getSourceLicense($source_name);
-            file_put_contents($target_dir . '/ext_' . $ext . '.txt', $content);
+            foreach ($this->getSourceLicenses($source_name) as $index => $license) {
+                file_put_contents("{$target_dir}/ext_{$ext}_{$index}.txt", $license);
+            }
```

Comment:
What do you think about adding a `"suffix"` or `"name"` field here? Sometimes it is possible to distinguish what LICENSE represents.

Like PHP contains PHP License and Zend Engine License, we could use:

 ```json
"license": [
            {
                "type": "file",
                "path": "LICENSE"
            },
            {
                "suffix": "zend",
                "type": "file",
                "path": "Zend/LICENSE"
            }
        ]
```

Result:

```
src_php-src_0.txt
src_php-src_zend.txt
```

Or do you have any better suggestions?

Reply:
I'm open to adding this but also I didn't want to make this PR too big 😉

Reply:
Not a problem, I can merge it first and then consider and update it later when checking the LICENSE for different libraries

Reply:
Here it goes: https://github.com/crazywhalecc/static-php-cli/pull/161